include/mvd/bits/sonata_misc.hpp

```diff
@@ -27,31 +27,56 @@
 
 #include "../sonata.hpp"
 
-namespace{
+
+namespace {
+
+using namespace bbp;
```

Comment:
This one I'm fine with, since it's used :)

Reply:
Got it. Thanks